apps/sms/index.html

```diff
@@ -128,7 +129,7 @@
 
   <body role="application" class="skin-comms theme-communications">
     <article id="main-wrapper" class="wrapper" data-position='right'>
-      <section id="thread-list" class="panel default-panel" role="region">
+      <section class="panel panel-thread-list" role="region">
```

Comment:
>  I remember we applied js- prefix for these classes. So 'panel-' prefix here also imply this class is not for styling but for navigation? 

I think that's what I instinctively had in mind. I don't know if you read http://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/ already, but this is the same idea: having prefixes to mean something more than just the class. Also now that I think a little more, what do you think about using a prefix "p-" instead of "panel-"?

> maybe we can start using panel-inbox/panel-conversation? Or even switch from panel to view like we do for folder structure and JS objects?

Yeah, I thought of it, but I thought it could be for a separate patch, because we need to change every place where isCurrentPanel and toPanel are used.